features/clip-path-animatable.yml

```
@@ -0,0 +1,6 @@
+name: Animatable clipping paths
+description: Clipping paths in CSS are animated using transitions and CSS animations.
+spec: https://drafts.fxtf.org/css-masking-1/#the-clip-path
```

The 'current' spec:
```suggestion
spec: https://www.w3.org/TR/css-masking/#the-clip-path
```

I would update the other spec links to match.

---

I'll defer this until resolution of https://github.com/web-platform-dx/web-features/issues/1785